Jenkinsfile

```diff
@@ -1,18 +1,26 @@
 #!groovy
-node('xnetPython01') {
+node('xnetPython') {
 
 	currentBuild.result = "SUCCESS"
 	
```

Comment:
Could you clean up the commit history?  @jashnani can show you how.  If it becomes a pain, I could always squash this change instead of doing a merge-commit.